### PR TITLE
Improve VS Code Marketplace show

### PR DESCRIPTION
**Pull request by atilafassina**
- [x] Change `displayName` to "Xata"
- [x] Bump version to `0.0.2`
- [x] Remove unparsed markdown from Settings pane
- [x] add absolute url to `icon`
- [x] Replace other images with permalink instead of branch related path
- [x] Add marketplace link to Readme

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "xata",
-  "displayName": "xata-vscode-extension",
+  "displayName": "Xata",
   "repository": "https://github.com/xataio/vscode-extension",
-  "icon": "doc/xata-icon-128.png",
+  "icon": "https://github.com/xataio/vscode-extension/raw/main/doc/xata-icon-128.png",
```
Comment (TejasQ) [inline package.json:5]: This can change if `main` changes. Use the permalink instead please (press <kbd>y</kbd> on the GitHub page of the file)

Comment (fabien0102) [reply package.json:5]: This need to be relative actually, vsce refuse to package an absolute link here 😅

Comment (atilafassina) [reply package.json:5]: Travis Kelce exploding head

Comment (atilafassina) [reply package.json:5]: > This need to be relative actually, vsce refuse to package an absolute link here 😅

my VSCode is squawking saying it needs an `https` url (literally that)

**Review by TejasQ — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (atilafassina) [reply package.json]: for the record:
@fabien0102 was right, VS Code was wrong.

**Pull request merged**